reactor-core/src/main/kotlin/reactor/util/function/TupleExtensions.kt

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (c) 2011-2017 Pivotal Software Inc, All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package reactor.util.function
+
+
+/**
+ * Extension for [Tuple2] to work with destructuring declarations.
+ *
+ * @author DoHyung Kim
+ * @since 3.1
+ */
+@Deprecated("To be removed in 3.3.0.RELEASE, replaced by module reactor-kotlin-extensions and import reactor.kotlin.core.util.function.component1",
```

Comment:
Since `ReplaceWith` suggests the imports, do we really need to duplicate it in the message?

Reply:
`ReplaceWith` suggestion is not offered with `operator` apparently, so encountering this one needs more manual guidance